readme.md

```diff
@@ -3,8 +3,10 @@
 
 ## Installation
 - Run `git clone https://github.com/strencher-kernel/pc-compat` inside your `packages` folder.
+- Run `cd pc-compat` once the repo has been cloned
+- Run `git switch -f dev` once you've entered the folder

```

Comment:
This shouldn't be a part of the installation instructions since it's not necessary to be on the dev branch. You can add it as a separate option though.

Reply:
The `cd pc-compat` can stay though

Reply:
Fair enough